terraform.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+#   cloud {
```

Comment:
Should we use `/*` and `*/` here instead?

Reply:
yes!